bugbug/utils.py

```diff
@@ -34,3 +35,14 @@ def fit(self, x, y=None):
 
     def transform(self, data):
         return np.array([elem[self.key] for elem in data]).reshape(-1, 1)
+
+
+class MissingOrdinalEncoder(OrdinalEncoder):
```

Comment:
Can you add a comment explaining that this is necessary until supported upstream in OrdinalEncoder?